package.json

```diff
@@ -58,7 +56,9 @@
     "babel-eslint": "9.0.0",
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.4",
+    "babel-preset-react-app": "^7.0.0",
```

Comment:
What does this do, and do we need it?

Reply:
https://github.com/ProjectMirador/mirador/blob/9d2a5f45a5add1b43475652b7bb1c73e72c7a176/config/webpack.config.dev.js#L165-L167
It provides some babel magic:
https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/webpack-overrides.js

Should be optional for the current build.